### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by williamito**
## Description of what I changed

Minor updates to README file and adds links to relevant wiki pages.

## E2E test

TESTED:

N/A

## Checklist: I completed these to help reviewers :)

- [ ] I have read and will follow the [review process](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/blob/master/doc/review_process.md).
- [ ] I am familiar with Google Style Guides for the language I have coded in.

  No? Please take some time and review [Java](https://google.github.io/styleguide/javaguide.html) and [Python](https://google.github.io/styleguide/pyguide.html) style guides.

- [ ] My IDE is configured to follow the Google [**code styles**](https://google.github.io/styleguide/).

  No? Unsure? -> [configure your IDE](https://github.com/google/google-java-format).

- [ ] I have **added tests** to cover my changes. (If you refactored existing code that was well tested you do not have to add tests)
- [ ] I ran `mvn clean package` right before creating this pull request and added all formatting changes to my commit.
- [ ] All new and existing **tests passed**.
- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ for a FHIR query.
 
 ![Modules involved in FHIR authorization/access-control](doc/summary.png)
```
Comment (bashir2) [inline README.md:21]: nit: now that you are at it, consider making this image smaller as well.

File: README.md [hunk 2]
```diff
@@ -51,8 +51,10 @@ java -Dloader.path="PATH-TO-ADDITIONAL-PLUGINGS/custom-plugins.jar" \
 ```
 
 The plugin library can be swapped with any third party access-checker as
-described in the [plugins](plugins) directory. Note Spring Boot is not a
-requirement for using the access-proxy; we just use it to simplify the
+described in the [plugins](plugins) directory and [the
+wiki](https://github.com/google/fhir-gateway/wiki/Understanding-access-checker-plugins).
+
+Note: Spring Boot is not a requirement for using the access-proxy; we just use it to simplify the
```
Comment (bashir2) [inline README.md:57]:
```suggestion
Note: Spring Boot is not a requirement for using the access-gateway; we just use it to simplify the
```

Comment (bashir2) [reply README.md:57]: It is probably worth clarifying that we use "proxy" and "gateway" interchangeably here.

**Review by bashir2 — APPROVED** [review]
Thanks @williamito just some minor comments.

File: README.md (hunk 2 above)
Comment (williamito) [reply README.md:57]: Added a note at the top, reworded a few cases, and updated some links from proxy to gateway.

**Review by bashir2 — APPROVED** [review]

**Comment by bashir2** [comment]
Merging as this is only documentation changes and the e2e was broken by the recent repo rename.

**Pull request merged**